src/utils/keystore.ts

```diff
@@ -153,10 +200,24 @@ export function getAddressFromPriv (secret) {
  * @param {Object} keyObject Keystore object.
  * @return {Buffer} Plaintext private key.
  */
-export async function recover (password, keyObject) {
+
+export async function recover (
+  password: string,
+  keyObject: {
+    id: string,
+    version: number,
+    public_key: string,
+    crypto: {
+      cipher_params: {nonce: string},
+      ciphertext: string,
+      symmetric_alg: string,
+      kdf: string,
+      kdf_params: Kdf_params
+    }
+  }) {
```

Comment:
We could create an interface regarding the keyObject such as 

```
interface SecretStorageKeyObject{
  id: string,
  version: number,
  public_key: string,
  crypto: {
    cipher_params: {nonce: string},
    ciphertext: string,
    symmetric_alg: string,
    kdf: string,
    kdf_params: Kdf_params
  }
}
```

and then use it in both `validateObject` and `recover` functions

```suggestion
export async function recover (
  password: string,
  keyObject: SecretStorageKeyObject) {
```